lib/index.js

```diff
@@ -15,9 +14,9 @@ var when = require('do-when');
 var AdWords = module.exports = integration('AdWords')
   .option('conversionId', '')
   .option('remarketing', false)
-  .option('whitelist', [])
-  .tag('<script src="//www.googleadservices.com/pagead/conversion_async.js">')
-  .mapping('events');
+  .option('whitelist', {})
```

Comment:
this should be a subsetting in the labelMap setting — i think that's better UX

Reply:
I feel like there would be too much noise in `labelMap` with this. Since whitelist is for events that you _don't_ want to map a label to, I think it should remain as a separate setting

Reply:
mmm yeah but think about it in practice. You'd have to flip back and forth to check what events you're mapping etc... which i guess that's what people have to do now. I think it'd be pretty nice to list the event in labelMap and then simply "check a box" to whitelist it so that it is also? 

Honestly having both of these top level settings seems unnecessary. 

```
Send Additional Remarketing Tags
If you enable this, we will send an additional remarketing tag for each of your conversion labels above and every .page() call!

Send Standalone Remarketing Tags
If you’d like to send standalone remarketing tags, you can whitelist your Segment Events here and we will send a remarketing tag for these events. If you already mapped an event as a conversion label above and have checked the option to send additional remarkeeting tags, you do not need to whitelist the event again.
```
These two settings seems like one off feature adds just patched on top of this integration... might be a good opportunity to consolidate but that would obv increase scope of this ticket...

Just seems like ALL of these could be under labelMap and keeps it on one UI sheet. I'd argue that's less complicated and just flows better and keeps context on the same page (prevents flipping back and forth between top level settings) and also I would not think it's too noisy.